node/node.go

```diff
@@ -138,6 +138,7 @@ func NewNode(ctx context.Context, conf config.NodeConfig, p2pKey crypto.PrivKey,
 		baseKV = store.NewDefaultKVStore(conf.RootDir, conf.DBPath, "dymint")
 	}
 	mainKV := store.NewPrefixKV(baseKV, mainPrefix)
+	// FIXME: not sure why dalcKV is needed. It's not used by celestia
```

Comment:
it's currently used by the mock but perhaps there is a more elegant way to transfer it. either way with current design it's needed.